src/styles/globals.ts

```diff
@@ -249,6 +249,9 @@ export const elGlobals = css`
       --font-weight-medium: 500;
       --font-weight-bold: 600;
 
+      /* elevation variables */
+      --elevation-xl: 0px 4px 16px 0px rgba(96, 120, 144, 0.16);
+
```

Comment:
I don't think the CSS elevation variable is declared before, so I put it in this file for now

Should I update the `tokens.json` and run the generator script? I am not sure

Reply:
**suggestion:** I would apply these styles locally to the Modal component for now. We'll need to discuss with Design how shadows should be handled and named in design tokens, as well as what zIndex elevated material should/should not have.